scripts/build-docker

```diff
@@ -1,5 +1,14 @@
 #!/usr/bin/env bash
 
+ARCH=$(go env GOARCH)
```

Comment:
this won't work for mastering builds from a single machine-- probably the way to do this is to parameterize the TAG and DOCKERFILE variables